pkg/webhook/mutation/pod_mutator/pod_mutator_test.go

```diff
@@ -284,7 +285,7 @@ func createTestWebhook(mutators []dtwebhook.PodMutator, objects []client.Object)
 }
 
 func createSimplePodMutatorMock() dtwebhook.PodMutator {
-	mutator := dtwebhook.PodMutatorMock{}
+	mutator := mocks.PodMutator{}
```

Comment:
`mocks.NewProvider(t)` should be used in create*Mock functions -> line [240](https://github.com/Dynatrace/dynatrace-operator/pull/2340/files#diff-130a5f5e79e744f15bb582d9ac4e4d97c3b8cd6722caeb0cc71054fb81e86ddbR240)

Reply:
I think we are mixing terms here! `NewProvider()` is an auto-generated function based on the config command interface, this doesn't exist on the webhook package I'm testing (AFAIK). IMO `mocks.PodMutator{}` is still the correct way, what do you think?

Reply:
Clarified in the call :+1: I just added the mock constructor call